pkg/controller/compliancescan/compliancescan_controller.go

```diff
@@ -286,7 +286,7 @@ func (r *ReconcileComplianceScan) phaseRunningHandler(instance *compv1alpha1.Com
 			}
 			if running {
 				// at least one pod is still running, just go back to the queue
-				return reconcile.Result{}, err
+				return reconcile.Result{Requeue: true, RequeueAfter: requeueAfterDefault}, err
```

Comment:
The commit msg mentions that the controller no longer listens to pod changes, but c.Watch() is still using Pod as a secondary

Reply:
I'll delete that. It tries to listen for pods (as long as their owner references are properly set), but none will be tracked as they now aren't owned by the compliancescan